src/prefect/utilities/logging.py

```diff
@@ -129,6 +129,13 @@ def emit(self, record) -> None:  # type: ignore
             assert isinstance(self.client, Client)  # mypy assert
 
             record_dict = record.__dict__.copy()
+
+            # ensures emitted logs respect configured logging level
+            config_level = getattr(logging, prefect.context.config.logging.level, 20)
```

Comment:
Could you use the logging enum instead of `20` here?

Reply:
Yup, done!